lldb/test/API/lang/swift/external_provider_extra_inhabitants/TestExternalProviderExtraInhabitants.py

```diff
@@ -0,0 +1,23 @@
+"""
+Test that the external provider calculates the extra inhabitants of clang types correctly
+"""
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+
+
+class TestExternalProviderExtraInhabitants(TestBase):
+
+    def setUp(self):
+        TestBase.setUp(self)
+
+    @skipUnlessDarwin
```

Comment:
What would it take to construct a non- Darwin-specific test?

Reply:
I now believe that the only heap allocated types that go through the external provider and types special-cased by the compiler. I don't know if we have any of those not on Darwin.